data/World/Overworld.json

```diff
@@ -240,6 +282,7 @@
         "scene": "Lost Woods",
         "hint": "the Lost Woods",
         "locations": {
+            "LW Under Boulder Blue Rupee": "here(can_blast_or_smash)",
```

Comment:
The use of here() is not incorrect but I can't think of a reason why a person might want to blow up this boulder, the come back as the other age to collect the rupee, so while the boulder does stay destroyed, using here seems like overkill.

I recommend adding a trick to collect this rupee through the boulder using the Boomerang.

Reply:
I'm not opposed to this boomerang trick. Could you please shed some light on how it works? Is it applicable to ALL boulders? Is there already some trick that would cover this?

Reply:
There are quite a few boomerang tricks that exist already, but none that would really cover this. Maybe call it something like "Fetch Rupees Inside Boulders with Boomerang", tooltip includes something like "applies to: Death Mountain Trail 2x, Lost Woods". You have to throw the boomerang to the side of the boulders so that it curves into them and picks up the rupee on the way back. The rupee hitbox is pretty small so it's not super easy to make it connect; takes me a few tries. Might have to aim downward a touch?